nodejs/week1/avg.js

```diff
@@ -0,0 +1,27 @@
+/*/create a file named avg.js which will calculate the 
+average of the numbers provided as command line arguments./*/
+const input = process.argv.slice(2).map(Number); 
+
+export default function getAverage(...numbers) { 
```

Comment:
No need to export this, it is fine to just have it as a function. Also, as you are passing an array, you can also omit the `...` operator.
```javascript
function getAverage(numbers) {
```